test/ir/ir-builder.spec.ts

```diff
@@ -733,6 +733,17 @@ describe("IRBuilder", () => {
     expect(ptrToInt).toBeInstanceOf(llvm.Value);
     expect(ptrToInt.name).toBe("name");
     expect(ptrToInt.type).toEqual(llvm.Type.getInt32Ty(context));
+
```

Comment:
Can we split this into two independent test cases. It makes it easier to understand what's broken in case there's an issue with either `createIntToPtr` or `createPtrToInt`. 

The following should give you a pointer type if I remember the API correctly: 

```
llvm.Type.getInt32Ty(context).getPointerTo()
```

Reply:
Okay... will do.

I'm also adding DebugInfo.h so I did not give this test much time.